roles/dns/templates/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM {{ base_docker_image }}
```

Comment:
There should be a default value for base_docker_image in roles/dns/defaults/main.yml:
```
base_docker_image: {{ 'centos:centos7' if openshift.common.deployment_type == 'origin' else 'rhel7' }}
```